src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentAuthorizationRequestTransaction.cs

```diff
@@ -1,29 +1,26 @@
-﻿using Newtonsoft.Json;
-
-namespace SwedbankPay.Sdk.Payments.CardPayments
-{
-    public class CardPaymentAuthorizationRequestTransaction
-    {
-        [JsonConstructor]
-        protected internal CardPaymentAuthorizationRequestTransaction(string cardNumber,
-                                                    int cardExpiryMonth,
-                                                    int cardExpiryYear,
-                                                    string cardVerificationCode = null,
-                                                    string cardHolderName = null)
-        {
-            CardNumber = cardNumber;
-            CardExpiryMonth = cardExpiryMonth;
-            CardExpiryYear = cardExpiryYear;
-            CardVerificationCode = cardVerificationCode;
-            CardHolderName = cardHolderName;
-        }
-
-
-        public int CardExpiryMonth { get; }
-        public int CardExpiryYear { get; }
-        public string CardHolderName { get; }
-
-        public string CardNumber { get; }
-        public string CardVerificationCode { get; }
-    }
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CardPaymentAuthorizationRequestTransaction
+    {
+        protected internal CardPaymentAuthorizationRequestTransaction(string cardNumber,
+                                                    int cardExpiryMonth,
+                                                    int cardExpiryYear,
+                                                    string cardVerificationCode = null,
+                                                    string cardHolderName = null)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)